students/terrance_jones/lesson6/test_mailroom_part4.py

```diff
@@ -0,0 +1,22 @@
+
+import mailroom_part4
+from mailroom_part4 import donor_list
+from mailroom_part4 import donor_exists
+from mailroom_part4 import add_or_update_donor
+from mailroom_part4 import thank_you_letter
```

Comment:
note that you can also do a one liner import on above:
`from mailroom_part4 import donor_list, donor_exists  # etc`